bowlingScore.test.js

```diff
@@ -0,0 +1,42 @@
+const BowlingGame = require('./bowlingScore')
+
+describe('bowlingGame', () => {
+  it('makes a bowling class', () => {
+    const bowlingGame = new BowlingGame();
+  })
+  it('han an empty array for frames', () => {
+    const bowlingGame = new BowlingGame();
+    expect(bowlingGame.frames).toEqual([])
+  })
+  // needs a double as don't want to actualy add a number to the array, just test it is being added
+  // it('roll once', () => {
+  //   const bowlingGame = new BowlingGame();
+  //   expect(bowlingGame.knockedPins(1)).toEqual(1)
+  // })
+  it('can roll a gutter game', () => {
+    const bowlingGame = new BowlingGame();
+    bowlingGame.knockedPins(0)
+    expect(bowlingGame.getScore()).toBe(0)
+    })
+
+  //   it('can roll a perfect game', () => {
+  //     const bowlingGame = new BowlingGame();
+  //     bowlingGame.knockedPins(10)
+  //     expect(bowlingGame.getScore()).toBe(300)
+    
```

Comment:
Even though commented out, this test would require strike scoring. So a test for a single strike may be easier first.